content/servicemesh/download.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Download and Install Istio CLI"
+date: 2018-11-13T16:36:43+09:00
+weight: 20
+draft: true
+---
+
+Before we can get started configuring Istio we’ll need to first install the command line tools that you will interact with. To do this run the following.
+
+```
+cd ~/environment
+
+curl -L https://git.io/getLatestIstio | sh -
+
+cd istio-1.0.x
+
+export PATH=$PWD/bin:$PATH
```

Comment:
Can we change the command to run from `/usr/local/bin` -- we do that for all the other commands we are installing... this would be more consistent, and would work in any future or existing terminals.

maybe `sudo mv -v bin/istioctl /usr/local/bin/` rather than the export command.

Reply:
fixed it to move the command into /usr/local/bin/